tsconfig.json

```diff
@@ -5,10 +5,14 @@
         "noImplicitAny": false,
         "sourceMap": false,
         "declaration": true,
-        "moduleResolution": "node"
+        "moduleResolution": "node", 
+        "typeRoots": [ "node_modules/@types" ]
     },
     "files": [
         "index.ts",
         "sample.ts"
+    ], 
+    "exclude": [
```

Comment:
Good idea, I did a test and compared master to my output for _build and they look the same. I think the node_modules are implicitly ignored so adding this is a no op but makes it more explicit in the config.